app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ActivitiesController < ApplicationController
```

Comment:
Could you create a suit test for this controller?
You can test the variables you're declaring.